gnomad/assessment/sanity_checks.py

```diff
@@ -190,3 +207,766 @@ def compare_row_counts(ht1: hl.Table, ht2: hl.Table) -> bool:
+    :param delimiter: String to use as delimiter when making group label combinations.
+    :param metric_first_label: If True, metric precedes label group, e.g. AC-afr-male. If False, label group precedes metric, afr-male-AC.
+    :return: None
+    """
+    t = t.rows() if isinstance(t, hl.MatrixTable) else t
+
+    field_check_expr = {}
+    field_check_details = {}
+    for subfield in ["AC", "AF"]:
+        # Check raw AC, AF > 0
+        check_field = f"{subfield}{delimiter}raw"
+        field_check_expr, field_check_details = make_field_check_dicts(
+            field_check_expr=field_check_expr,
+            field_check_details=field_check_details,
+            check_description=f"{check_field} > 0",
+            cond_expr=t.info[check_field] <= 0,
+            display_fields=hl.struct(**{check_field: t.info[check_field]}),
+        )
+
+        check_field = f"{subfield}{delimiter}adj"
+        field_check_expr, field_check_details = make_field_check_dicts(
+            field_check_expr=field_check_expr,
+            field_check_details=field_check_details,
+            check_description=f"{check_field} >= 0",
+            cond_expr=t.info[check_field] < 0,
+            display_fields=hl.struct(
+                **{check_field: t.info[check_field], "filters": t.filters}
+            ),
+        )
+
+    # Check raw AN > 0
+    check_field = f"AN{delimiter}raw"
+    field_check_expr, field_check_details = make_field_check_dicts(
+        field_check_expr=field_check_expr,
+        field_check_details=field_check_details,
+        check_description=f"{check_field} > 0",
+        cond_expr=t.info[check_field] <= 0,
+        display_fields=hl.struct(**{check_field: t.info[check_field]}),
+    )
+
+    # Check adj AN >= 0
+    check_field = f"AN{delimiter}adj"
+    field_check_expr, field_check_details = make_field_check_dicts(
+        field_check_expr=field_check_expr,
+        field_check_details=field_check_details,
+        check_description=f"{check_field} >= 0",
+        cond_expr=t.info[check_field] < 0,
+        display_fields=hl.struct(**{check_field: t.info[check_field]}),
+    )
+    # Check overall gnomad's raw subfields >= adj
+    for subfield in ["AC", "AN", "nhomalt"]:
+        check_field_left = f"{subfield}{delimiter}raw"
+        check_field_right = f"{subfield}{delimiter}adj"
+        field_check_expr, field_check_details = make_field_check_dicts(
+            field_check_expr=field_check_expr,
+            field_check_details=field_check_details,
+            check_description=f"{check_field_left} >= {check_field_right}",
+            cond_expr=t.info[check_field_left] < t.info[check_field_right],
+            display_fields=hl.struct(
+                **{
+                    check_field_left: t.info[check_field_left],
+                    check_field_right: t.info[check_field_right],
+                }
+            ),
+        )
```

Comment:
PART OF LARGE SUGGESTION
```suggestion

        field_check_expr[f"{check_field_left} >= {check_field_right}"] = {
            "expr": hl.agg.count_where(
                t.info[check_field_left] < t.info[check_field_right]
            ),
            "display_fields": hl.struct(
                **{
                    check_field_left: t.info[check_field_left],
                    check_field_right: t.info[check_field_right],
                }
            ),
        }
```